sn_cli/src/acc_packet.rs

```diff
@@ -109,7 +104,7 @@ impl AccountPacket {
     /// when the user requests to encrypt data, the provided Client's signer key is used.
     pub fn init(
         client: Client,
-        wallet_dir: &Path,
+        root_dir: &Path,
```

Comment:
Let's keep this to be the `wallet_dir` since that's what it is, root dir doesn't mean anything in this context. The acc-packet currently needs a dir where to load the wallet from, it doesn't use it for any other thing.